MungPlex/Log.cpp

```diff
@@ -129,4 +131,44 @@ void MungPlex::Log::LogInformation(const char* text, const bool appendToLast, co
 void MungPlex::Log::LogInformation(const std::string& text, const bool appendToLast, const int indentation)
 {
 	LogInformation(text.c_str(), appendToLast);
-}
\ No newline at end of file
+}
+
+// Accepts as many const char* arguments as you want, but it is highly recommended (Needed) that you use the same number of arguments as the LogMessage you choose corresponds to, or number of %s's.
+// Messing up will result in garbage data being read, and potentially with some luck, the BEL sound being logged! ;)  
+void MungPlex::Log::LogInformation(MungPlex::LogMessages::LogMessageIntegers _Enum, ...) {
```

Comment:
Instead, consider implementing a logging library such as [`spdlog`](https://github.com/gabime/spdlog)(`vcpkg install spdlog`)

Reply:
I noticed this last night after I was checking the Pr myself.  I was afraid to straight up delete the file because I thought it might delete it locally.  I’m not too familiar with git, so it would nice if you could tell me how I should resolve this. Thanks!

Reply:
Do we want to add another library?

Reply:
Why not? It's a relatively small library. I would add it, good logging is always important for a well-developed app 😆

Reply:
Alright! I’ll add it I guess